dev-util/radare2/radare2-2.2.0.recipe

```diff
@@ -30,7 +30,7 @@ PROVIDES="
 	cmd:rarun2$secondaryArchSuffix
 	cmd:rasm2$secondaryArchSuffix
 	cmd:rax2$secondaryArchSuffix
-	lib:libr2$secondaryArchSuffix = 2.0.1
+	lib:libr2$secondaryArchSuffix = 2.2.0
```

Comment:
Shall we insert `libVersion` and `libVersionCompat` in this recipe?
```diff
 ARCHITECTURES="?x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
```
This will allow to have smaller diffs for the next updates.
```diff
-	lib:libr2$secondaryArchSuffix = 2.2.0
+	lib:libr2$secondaryArchSuffix = $libVersionCompat
```